sql/moz-fx-data-shared-prod/firefox_ios_derived/new_profile_activation_v1/query.sql

```diff
@@ -70,8 +70,11 @@ SELECT
   country,
   first_seen_date,
   submission_date,
-  1 AS new_profile,
-  CAST(days_2_7 > 1 AND COALESCE(search_count, 0) > 0 AS integer) AS activated
+  1 AS new_profile,  -- # TODO: should this be a boolean value? Also, if this is always 1 should we just assume an entry in this table means new_profile?
```

Comment:
I strongly prefer booleans. Counting is then `COUNTIF(is_new_profile)`, over `SUM(new_profile)`

Reply:
What is your reasoning for the boolean preference?
In terms of understanding the column as a metric instead of a dimension, I have a preference for numeric fields. It also makes it more straightforward to add to Looker.

Reply:
Definitely personal preference rules here. But consistency matters, and we have largely used boolean over numeric. There's a few ancillary benefits (less storage overhead, schema limits to yes/no/null, easier to understand what the field represents), but in general we should stick with one approach for all of our tables.

Reply:
So here we can remove the new_profile column, as it was only intended as a shortcut to make it a metric and avoid people having to check if this table only contained new profiles, and convert activated to is_activated.

Reply:
@kik-kik you should confirm this with the AERO team. Not sure if ya'll have standards / guidance for data modeling that we can reference, but it would be helpful for situations like this

Reply:
For added context, another reason I initially chose numeric was that this table was replacing https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/firefox_nondesktop_day_2_7_activation_v1/query.sql#L8-L12 - which was pre-aggregated and therefore numeric and queried with SUM, but yes it'd be great to hear what other people think and make an official standard out of it.